arm-recoveryservices/2016-06-01/swagger/vaultusages.json

```diff
@@ -0,0 +1,188 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2016-06-01",
+    "title": "RecoveryServicesClient",
+    "x-ms-code-generation-settings": {
+      "internalConstructors": false
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/usages": {

```

Comment:
After some offline discussions it seems exposing the operations API as-is would be the right thing to do. 
Alternatively:
- You could expose all the right APIs in the swagger documents for completeness OR
- You could return only those APIs that are currently in swagger from the operations API
@ravbhatnagar could you chime in here?